util/src/kvdb.rs

```diff
@@ -123,16 +126,65 @@ pub struct CompactionProfile {
 impl Default for CompactionProfile {
 	/// Default profile suitable for most storage
 	fn default() -> CompactionProfile {
+		CompactionProfile::ssd()
+	}
+}
+
+/// Given output of df command return Linux rotational flag file path.
+pub fn rotational_from_df_output(df_out: Vec<u8>) -> Option<PathBuf> {
```

Comment:
Shouldn't this be `#[cfg(linux)]` or some such? Might not need to be public either.